ui/org.eclipse.pde.launching/src/org/eclipse/pde/internal/launching/JUnitLaunchRequirements.java

```diff
@@ -0,0 +1,163 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Advantest and others.
```

Comment:
@iloveeclipse I'm not sure what we write here, we are moving code and then adjusting it. I believe the original code was added by @HannesWell, the original files were from IBM (though not the moved code, I think).